src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/controllers/AttachExceptionRecordWithOcrTest.java

```diff
@@ -241,6 +259,7 @@ void should_fail_correctly_if_ccd_is_down() throws Exception {
         caseData.put(JOURNEY_CLASSIFICATION, SUPPLEMENTARY_EVIDENCE_WITH_OCR);
         caseData.put("formType", "B123");
         caseData.put(SEARCH_CASE_REFERENCE_FIELD_NAME, CASE_ID);
+        caseData.put(ENVELOPE_ID_FIELD_NAME, "2777a98c-cbb4-11ea-87d0-0242ac130003");
```

Comment:
would be good to have a comment where this value is coming from or smth similar for future linking

Reply:
by the looks after scrolling all this there are different envelope ids here and this is more generic method: `getCaseData`

Reply:
This change only extends the existing code by the required field which was absent in the request so far.

The method looks generic, but it isn't - it returns case data expected in the response to callback request from `valid-supplementary-evidence-with-ocr.json`. Method and several variables need renaming, comments need to be added, but that's outside the scope of this PR. I'll create another one for just that.